src/Components/MovieForm/MovieForm.js

```diff
@@ -0,0 +1,97 @@
+import React, {useState} from 'react';
+import {GenreListArray} from '../../data';
+import styles from './MovieForm.module.scss';
+
+const MovieForm = ({formContent, action}) => {
+    formContent = action === 'edit' || action === 'delete' ? formContent : {};

```

Comment:
The MovieForm component is used for creating/editing a movie, right? In that case, weirdly, the component knows about the action 'delete'.